x/gov/types/genesis.go

```diff
@@ -50,6 +50,18 @@ func ValidateGenesis(data *GenesisState) error {
 			threshold.String())
 	}
 
+	expeditedThreshold := data.TallyParams.ExpeditedThreshold
+	if expeditedThreshold.IsNegative() || expeditedThreshold.GT(sdk.OneDec()) {
+		return fmt.Errorf("governance expedited vote threshold should be positive and less or equal to one, is %s",
+			expeditedThreshold)
+	}
+
+	if expeditedThreshold.LT(threshold) {
```

Comment:
```suggestion
	if expeditedThreshold.LTE(threshold) {
```

Reply:
I will update locally because err messages need to be updated as well